nncf/quantization/algorithms/bias_correction/algorithm.py

```diff
@@ -446,7 +447,8 @@ def output_filter_func(point):
     def get_statistic_points(self, model: TModel) -> StatisticPointsContainer:
         self._set_backend_entity(model)
         model_copy = self._remove_fq_from_inputs(copy_model(model))
-        nncf_graph = NNCFGraphFactory.create(model_copy) if self.nncf_graph is None else self.nncf_graph
+        model_copy = self._insert_null_biases(model_copy)
```

Comment:
Is it applicable for the fast bias correction algorithm?

Reply:
Yes, applicable. We just need to provide the same method for the FastBC algorithm.

Reply:
Are you going to do it in this PR or in a follow-up PR?

Reply:
Since we need to have green conformance, I think that it would be better to have in this PR to avoid double validation.